Makefile

```diff
@@ -81,7 +81,7 @@ tool-install:
 
 lint: tool-install $(FFMPEG_BUILD)
 	go mod tidy
-	CGO_CFLAGS=$(CGO_CFLAGS) GOFLAGS=$(GOFLAGS) $(TOOL_BIN)/golangci-lint run -v --fix --config=./etc/.golangci.yaml
+	CGO_CFLAGS=$(CGO_CFLAGS) GOFLAGS=$(GOFLAGS) $(TOOL_BIN)/golangci-lint run -v --fix --config=./etc/.golangci.yaml --timeout=2m
```

Comment:
curious were you running into hangs on lint?

Reply:
Yeah not sure why it is taking so much longer now...

https://github.com/viam-modules/video-store/actions/runs/12162796386/job/33920591465